portfolio/src/main/webapp/index.html

```diff
@@ -55,12 +56,12 @@ <h3>More coming soon!</h3>
           </div>
         </div>
       </div>
-      <div id="contact-cotainer">
+      <div id="contact-container">
         <div id="contact">
-          <h2><span class="highlight" target="_blank">Contacts</span></h2>
-          <h3>Github: <a href="https://github.com/Froggerson" target="_blank">https://github.com/Froggerson</a></h3>
-          <h3>Email: <a href="phamjennifer314@gmail.com">phamjennifer314@gmail.com</a></h3>
-          <h3>Linkedin: <a href="https://www.linkedin.com/in/jennifer-pham-575ab113b" target="_blank" >https://www.linkedin.com/in/jennifer-pham-575ab113b</a></h3>
+          <h3>Contact me</h3>
+          <a href="https://github.com/Froggerson" target="_blank"><img width="32" height="32" src="/images/github.png"/></a>
+          <a href="phamjennifer314@gmail.com" target="_blank"><i class="material-icons md-32">email</i></a>
```

Comment:
Have you considered using a mailto link here? https://en.wikipedia.org/wiki/Mailto

Reply:
done!